docs/docs/en/guide/task/flink.md

```diff
@@ -24,7 +24,7 @@ Flink task type for executing Flink programs. For Flink nodes, the worker submit
 - **Program type**: Supports Java, Scala and Python.
 - **The class of main function**: The **full path** of Main Class, the entry point of the Flink program.
 - **Main jar package**: The jar package of the Flink program (upload by Resource Center).
-- **Deployment mode**: Support 2 deployment modes: cluster and local.
+- **Deployment mode**: Support 3 deployment modes: cluster, local and application.
```

Comment:
It is better to add an example about how to using flink in `application` mode

Reply:
Er.. The only difference is the run command. Should I add some explaination for those 3 modes or simply add a reference link to Flink official website?

Reply:
I think adding some links to FL website is better, but please use the fix version docs and avoid using `latest` or `stale` url